Source/SwiftLintFramework/Rules/SortedImportsRule.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  SortedImportsRule.swift
+//  SwiftLint
+//
+//  Created by Scott Berrevoets on 12/15/16.
+//  Copyright © 2016 Realm. All rights reserved.
+//
+
+import SourceKittenFramework
+
+public struct SortedImportsRule: ConfigurationProviderRule, OptInRule {
+    public var configuration = SeverityConfiguration(.warning)
+
+    public init() {}
+
+    public static let description = RuleDescription(
+        identifier: "sorted_imports",
+        name: "Sorted Imports",
+        description: "Imports should be sorted.",
+        nonTriggeringExamples: [
+            "import AAA\nimport BBB\nimport CCC\nimport DDD"
+        ],
+        triggeringExamples: [
+            "import AAA\nimport ZZZ\nimport BBB\nimport CCC",
+            "import AAA\nimport ZZZ\nimport BBB\nimport CCC\nimport AAA"
+        ]
+    )
+
+    public func validateFile(_ file: File) -> [StyleViolation] {
+        let pattern = "import\\s+(\\w+)"
+        let excludingKinds = SyntaxKind.commentAndStringKinds()
```

Comment:
instead of _excluding_ comments and strings, you should be matching a single keyword followed by a single identifier, as highlighted by getting the syntax information from imports:

```shell
$ sourcekitten syntax --text "import AAA"                
[
  {
    "type" : "source.lang.swift.syntaxtype.keyword",
    "offset" : 0,
    "length" : 6
  },
  {
    "type" : "source.lang.swift.syntaxtype.identifier",
    "offset" : 7,
    "length" : 3
  }
]
```